spec/delete_host_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'rails_helper' # spec_helper?
+require 'sidekiq/testing'
+
+describe DeleteHost do
+  let(:user) { FactoryBot.create(:user) }
+  let(:host) { FactoryBot.create(:host, org_id: user.account.org_id) }
+  let(:profile) { FactoryBot.create(:profile, :with_rules, account: user.account) }
+  let(:message) { { 'id' => host.id, 'type' => 'delete' } }
```

Comment:
I think you can extract the `id` part into a next `let` block under the name of `message_id` and then just set that in various contexts accordingly:
```
  let(:message) { { 'id' => message_id, 'type' => 'delete' } }
  let(:message_id) { 'something' }

  context 'something else' do
    let(:message_id) { 'something else' }
    ...
  end
```